aesara/tensor/slinalg.py

```diff
@@ -39,10 +40,9 @@ class Cholesky(Op):
     # TODO: for specific dtypes
     # TODO: LAPACK wrapper with in-place behavior, for solve also
 
-    __props__ = ("lower", "destructive", "on_error")
+    __props__ = ("destructive", "on_error")
```

Comment:
Do we need `destructive` attribute here? 

According to the [Op docs](https://aesara.readthedocs.io/en/latest/extending/op.html#props__), `__props__`  lists the attributes which influence the computation performed. And I am not sure how `destructive` attribute is used in `perform` method.

Reply:
My guess is that this was added for some functionality that was never implemented (e.g., a faster method that alters the input variables in place). Looking around the library I don't think this is being used anywhere so maybe we can remove it.